### DISCUSSION TITLE
Ark-SA shop idea

### DISCUSSION THREAD

**Discussion by PoeBloodstone** [gameserverapp/Platform#919] [Suggestions]
I know shops are a chalange and this may not be able to be done, but idea. Seeing we have no tokens or things like that, Make a opthion to select a currancy ingame item. 

Example, we can make shop packages say deliver dodo but we can only set it to be 100 **tokens** witch for ArkSa tokens dont exist cus, shop = bad thing for cross play. 
But
If there was a button to add blueprint path for ingame thing as the currancy we could make them sell for like wood, or element/any item in game.  Then people could make bushberrie shops that are fully not bought ingame items so i would not think  ageinst the "no shops for real $$ thin"g console prevents us from doing.

**Comment by WEchoW** [comment]
I think Max will be never able to do this on GSA for ASA servers, it doesn't sound legit. It's like you tell policeman you sell apples, but actually inside of this apples is illegal weed.

**Comment by mvaessen** [comment]
Hi, thanks for your suggestion.
Since this is very specific to ARK, and hard to replicate on other games, is this not something I'm planning to add. 

And I agree with @WEchoW that this would most likely be considered a way to work around CurseForge's shop policy.
